python/satyaml/ERMINAZ-1U.yml

```diff
@@ -12,3 +13,17 @@ transmitters:
     framing: QUBIK
     data:
     - *tlm
+  # New framing:
+  9k6 FSK CCSDS TMTC SPP downlink:
+    frequency: 435.775e+6
+    modulation: FSK
+    baudrate: 9600
+    deviation: 4800
+    framing: CCSDS Reed-Solomon
+    frame size: 128
+    RS basis: conventional
+    RS interleave depth: 1
```

Comment:
It is not necessary to specify the `RS interleave depth`, since the default is one.

Reply:
Fixed (removed).

Moreover, the correct keyword to specify a non-default R-S interleaving depth would have been `RS interleaving`.